gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -210,18 +210,18 @@ private static void configureErrorProneOptions(
                 .set(String.format(
                         "%s%s(build|src%sgenerated.*)%s.*",
                         Pattern.quote(projectPath), separator, separator, separator));
-        errorProneOptions.check("CatchSpecificity", CheckSeverity.OFF);
-        errorProneOptions.check("UnusedVariable", CheckSeverity.OFF);
-        errorProneOptions.check("EqualsHashCode", CheckSeverity.ERROR);
-        errorProneOptions.check("EqualsIncompatibleType", CheckSeverity.ERROR);
-        errorProneOptions.check("StreamResourceLeak", CheckSeverity.ERROR);
-        errorProneOptions.check("InputStreamSlowMultibyteRead", CheckSeverity.ERROR);
-        errorProneOptions.check("JavaDurationGetSecondsGetNano", CheckSeverity.ERROR);
-        errorProneOptions.check("URLEqualsHashCode", CheckSeverity.ERROR);
 
+        errorProneOptions.disable("UnnecessaryParentheses", "CatchSpecificity", "UnusedVariable");
```

Comment:
I've found UnnecessaryParentheses useful in a few places, thoughts on disabling it only on jdk14 with experimental features enabled?

Reply:
Switch expressions aren't an experimental feature in Java 14.